src/inputs/checkbox/checkbox.jsx

```diff
@@ -11,14 +11,23 @@ const propTypes = {
     checked: PropTypes.bool,
     className: PropTypes.string,
     /**
-     * A Checkbox can be disabled.
+     * Deprecated prop. Please use `disabled` instead.
      */
     disable: PropTypes.bool,
     /**
-     * Deprecated prop. Please use `disable` instead.
+     * A Checkbox can be disabled.
      */
     disabled: PropTypes.bool,
     fluid: PropTypes.bool,
+    /**
+     * Forwarded Ref
+     */
+    forwardedRef: PropTypes.oneOfType([
+        // Either a function
+        PropTypes.func,
+        // Or the instance of a DOM native element (see the note about SSR)
```

Comment:
Did I put `(see the note about SSR)`?  I wonder why?  I must have copied that from somewhere.  I have _NO IDEA_ what it means! =)

Reply:
Yeah, if you would be so kind as to remove all of these, that'd be great.  I think it's clear without that.  It's either a function (like a component) or it's a native DOM element.  Period.  Thanks!  And sorry about that.

Reply:
OOOH ... I think SSR = "server-side rendering"!
Which, we don't do, so it's not applicable.